desktop/apps/article2/components/App.js

```diff
@@ -28,21 +27,29 @@ export default class App extends React.Component {
   }
 
   renderArticle = () => {
-    const article = this.props.article
-    if (article.layout === 'standard' && !this.props.isSuper) {
-      const emailSignupUrl = this.props.subscribed ? '' : `${sd.APP_URL}/signup/editorial`
+    const { article, isSuper, subscribed } = this.props
+
+    if (article.layout === 'standard' && !isSuper) {
+      const emailSignupUrl = subscribed ? '' : `${sd.APP_URL}/signup/editorial`
 
       return (
-        <InfiniteScrollArticle article={article} emailSignupUrl={emailSignupUrl} />
+        <InfiniteScrollArticle
+          article={article}
+          emailSignupUrl={emailSignupUrl}
+        />
       )
     } else {
-      const navHeight = this.props.isSuper ? '0px' : NAVHEIGHT
+      const navHeight = isSuper ? '0px' : NAVHEIGHT
+      const articleMarginTop = article.layout === 'standard' ? '100' : '0'
+      const headerHeight = `calc(100vh - ${navHeight})`
 
       return (
         <Article
           article={article}
-          headerHeight={`calc(100vh - ${navHeight})`}
-          marginTop={article.layout === 'standard' ? '100' : '0'}
+          relatedArticlesForPanel={article.relatedArticlesPanel}
+          relatedArticlesForCanvas={article.relatedArticlesCanvas}
+          headerHeight={headerHeight}
```

Comment:
I think we might only want the panel articles here -- super-articles have their own footer with their sub-articles, and are excluded from infinite scroll. 
@owendodd do super-articles have the related canvas at the bottom?

Reply:
@eessex no they shouldn't as they are almost always brand partnerships

Reply:
we'll have to revisit this in the future, but that's the most straightforward answer for the time being